lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -0,0 +1,115 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for the use of Time.now instead of Time.zone.now etc
+      # Built on top of this article http://danilenko.org/2012/7/6/rails_timezones/
```

Comment:
There's also a section about this in the Rails style guide. I'd suggest extending it, as RuboCop is generally based on the Ruby & Rails style guides. 

I'd also suggest adding some concrete code examples in the class doc itself.